server/athenian/api/experiments/preloading/entries.py

```diff
@@ -49,7 +49,6 @@
     set_matched_by_from_release_match,
```

Comment:
FYI: the code inside `experiments` is legacy. We should keep it as-is. It's not used anywhere.